Gemfile

```diff
@@ -72,3 +72,12 @@ group :test do
 end
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍 Just like I have made my logo which you can see in the image above ⬆️

Reply:
@HAMMAS-SALEEM  hello i can't see your logo can reshare it with me

Reply:
@ilyassatlassi I hope you are doing well. I suggested you make your own logo which you can put in your readme file.